File: Learn_Android/Old/aboutActivity/app/src/main/java/com/example/sunkai/aboutactivity/MainActivity.java
package com.example.sunkai.aboutactivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

public class MainActivity extends BaseActivity {
    private Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        button=findViewById(R.id.button1);
        button.setOnClickListener(view -> {
            Intent intent=new Intent(MainActivity.this,Main2Activity.class);
            startActivity(intent);
        });
    }
}

File: Learn_Android/ActivityProxyHost/app/src/main/java/com/example/sunkai/activityproxyhost/myapplicaiton/MainActivity.kt
package com.example.sunkai.activityproxyhost.myapplicaiton

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.os.Environment
import android.widget.Toast
import com.example.sunkai.activityproxyhost.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Toast.makeText(this,Environment.getExternalStorageDirectory().absolutePath,Toast.LENGTH_LONG).show()
        btn_reflect.setOnClickListener {
            val intent=Intent(this,ProxyActivity::class.java)
            startActivity(intent)
        }

        btn_interface.setOnClickListener{
            val intent=Intent(this,PlugProxyActivity::class.java)
            startActivity(intent)
        }
    }
}
